src/components/Accordion/Accordion/Accordion.jsx

```diff
@@ -33,9 +33,12 @@ const Accordion = forwardRef(({ children: originalChildren, allowMultiple, defau
   const onChildClick = useCallback(
     itemIndex => {
       if (allowMultiple) {
-        const newExpandedItems = expandedItems;
+        const newExpandedItems = [...expandedItems];
```

Comment:
the first bug was that changing the state does not cause a render because the state was mutable